### DISCUSSION TITLE
安装失败，应用安装程序无法安装包相关项

### DISCUSSION THREAD

**Discussion by TonyStarkzy** [ywmoyue/biliuwp-lite#895] [General]
### Bug 描述

更新总是无法安装，卸载重装也不行，win11专业版

### 操作步骤

```plain text
解压缩
安装证书
点击安装
安装失败，显示应用安装程序无法安装包相关项
```

### 期望结果

希望能正常安装

### 截图

![bilib](https://github.com/user-attachments/assets/37a835d7-1607-4a74-8fa5-3e65e8707233)

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

win11-24h2

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
不是bug，转discussion

**Comment by ywmoyue** [comment]
@TonyStarkzy 看下其他人的相同问题：https://github.com/ywmoyue/biliuwp-lite/discussions/855

**Reply by TonyStarkzy** [reply]
你好，我在设置中开启了开发者选项，但是管理员终端执行命令 get-ExecutionPolicy后输出 Restricted怎么办呢
![image](https://github.com/user-attachments/assets/b0ecbc1e-4676-4f53-8325-9fc03a34de3d)

**Reply by ywmoyue** [reply]
@TonyStarkzy 照着安装教程继续下一步，下一步就是启用脚本执行权限

**Reply by ywmoyue** [reply]
![image](https://github.com/user-attachments/assets/2f01d0bd-a4b7-4021-846c-cfedb100cac4)

**Reply by TonyStarkzy** [reply]
非常感谢，已经安装成功了，谢谢！！！
